src/components/menus/BottomMenu.js

```diff
@@ -93,15 +96,21 @@ export const BottomMenu = ({ menu }) => {
                 <ul className={`list-no-style ${sty.SocialLinks}`}>
                   {menu.socials.map((item, index) => (
                     <li>
-                      <PrismicLink href={'/'} key={`socialLink:${index}`}>
+                      <PrismicLink
+                        href={'/'}
+                        key={`socialLink:${index}`}
+                        title={item.social_icon?.alt}
```

Comment:
Not sure what should be title in this case, alt text doesn't seem great since it refers to the image not the link but the url could be totally undescriptive also
![image](https://github.com/user-attachments/assets/dedb5c28-6a5a-45f9-a764-708ee0f0cc73)

Reply:
In these cases what I generally do when the image is decoration for the link, is I would set the title to "Link to _____" so it's clear to the screenreader. You're correct that the icon doesn't really matter that much so much as what it represents here.